tests/test_repo.py

```diff
@@ -3,164 +3,53 @@
 import subprocess
 from pathlib import Path
 
-import toml
-import pytest
-from aea.cli.utils.config import get_default_author_from_cli_config
 
-from auto_dev.utils import change_dir
-
-
-class BaseTestRepo:
-    """Test scaffolding new repository."""
-
-    repo_name = "dummy"
-    type_of_repo: str
-    make_commands: tuple[str]
-
-    @property
-    def cli_args(self):
-        """CLI arguments."""
-        return ["adev", "repo", "scaffold", self.repo_name, "-t", self.type_of_repo]
-
-    @property
-    def parent_dir(self):
-        """Parent directory of newly scaffolded repo."""
-        return Path.cwd()
-
-    @property
-    def repo_path(self):
-        """Path of newly scaffolded repo."""
-        return self.parent_dir / self.repo_name
```

Comment:
I get a little emotional seeing all of this. Reminds me of my boyfriend, whose name I won't mention, so as not to use the N-word, who wrote: *"The weak and the botched shall perish: first principle of our charity. And one should help them to it."*

RIP to that cursed base class